scripts/Release-Test-Script-Platform-Overview.md

```diff
@@ -0,0 +1,63 @@
+# Overview for release-test-script-platform.sh
```

Comment:
this is in the wrong place... This is related to ut-control and not ut-core.

You need to move it to ut-control, alone with the release script for ut-control.

Reply:
ut-control test script can be found here :  https://github.com/rdkcentral/ut-control/pull/46/files

In this PR we have 2 test scripts:
release-test-script-platform.sh : Tests changes wrt to platform hal. Changes are tested against platform hal repo eg: rdk-halif-hdmi_cec
release-test-script-ut-core.sh : Tests changes wrt ut-core. Changes are tested against ut-core repo.